rich/_inspect.py

```diff
@@ -233,3 +233,38 @@ def _get_formatted_doc(self, object_: Any) -> Optional[str]:
         if not self.help:
             docs = _first_paragraph(docs)
         return escape_control_codes(docs)
+
+
+def object_types_mro(obj: Union[object, Type[Any]]) -> Tuple[type, ...]:
+    """Returns the MRO of an object's class, or of the object itself if it's a class."""
+    obj_type: Type[Any] = type(obj)
+    if obj_type is type:
+        # the given object was already a type: let's use this instead:
+        obj_type = cast(Type[Any], obj)  # I wish MyPy could infer this itself :-/
+    return getattr(obj_type, "__mro__", ())
+
+
+def object_types_mro_as_strings(obj: object) -> Collection[str]:
```

Comment:
`get_object_imports` ?

Reply:
These are not imports: it's the list of an object's class (or the object itself it's a class) hierarchy in terms of OOP :slightly_smiling_face: 

So `get_object_types_mro_as_strings(TotalFileSizeColumn())` for example returns `['rich.progress.TotalFileSizeColumn', 'rich.progress.ProgressColumn', 'abc.ABC', 'builtins.object']`

Maybe it could be named `get_object_hierarchy_as_strings`, or something like this?